tools/ci_connector_ops/ci_connector_ops/qa_engine/constants.py

```diff
@@ -20,12 +20,19 @@
     f"https://github.com/{AIRBYTE_PLATFORM_INTERNAL_REPO_OWNER}/{AIRBYTE_PLATFORM_INTERNAL_REPO_NAME}.git"
 )
 AIRBYTE_PLATFORM_INTERNAL_MAIN_BRANCH_NAME = "master"
-AIRBYTE_PLATFORM_INTERNAL_PR_ENDPOINT = (
-    f"https://api.github.com/repos/{AIRBYTE_PLATFORM_INTERNAL_REPO_OWNER}/{AIRBYTE_PLATFORM_INTERNAL_REPO_NAME}/pulls"
+AIRBYTE_PLATFORM_INTERNAL_REPO_ENDPOINT = (
+    f"https://api.github.com/repos/{AIRBYTE_PLATFORM_INTERNAL_REPO_OWNER}/{AIRBYTE_PLATFORM_INTERNAL_REPO_NAME}"
 )
+AIRBYTE_PLATFORM_INTERNAL_PR_ENDPOINT = f"{AIRBYTE_PLATFORM_INTERNAL_REPO_ENDPOINT}/pulls"
+AIRBYTE_PLATFORM_INTERNAL_ISSUES_ENDPOINT = f"{AIRBYTE_PLATFORM_INTERNAL_REPO_ENDPOINT}/issues"
+
 GITHUB_API_TOKEN = os.environ.get("GITHUB_API_TOKEN")
 GITHUB_API_COMMON_HEADERS = {
     "Accept": "application/vnd.github+json",
     "X-GitHub-Api-Version": "2022-11-28",
     "Authorization": f"Bearer {GITHUB_API_TOKEN}",
 }
+GIT_USERNAME_FOR_AUTH = "octavia-squidington-iii"
+GIT_USER_EMAIL = f"{GIT_USERNAME_FOR_AUTH}@sers.noreply.github.com"
+GIT_USERNAME = "Octavia Squidington III"
+PR_LABELS = ["team/connector-ops", "cloud-availability-updater"]
```

Comment:
Do we have a plan in place to monitor these labels? Or would it be better if we were requested as reviewers automatically?